### PR TITLE
add color customizations for inline debug values

### LINKED ISSUE #120936: Inline Values Should be Themable
The inlines values colors are currently hardcoded:
https://github.com/microsoft/vscode/blob/1ed73408cd2a9229dfd67de33ccd8a3a801bdcd5/src/vs/workbench/contrib/debug/browser/debugEditorContribution.ts#L74-L88

These should be abstracted to a themable color, either its own dedicated color or one of the [Testing View Colors](https://code.visualstudio.com/api/references/theme-color#testing-view-colors), I would recommend hint or info.

### PR DESCRIPTION
**Pull request by nrayburn-tech**
Modifies inline debug values to be allow color customizations/themes.  Defaults should still be what they were previously, just converted to hex from rgba.

New color ids, `debugInline.decorationBackground` and `debugInline.decorationForeground`.  The names may need to be changed to better align with the other ids.

Example of modified colors.
Screen Shot 2021-05-05 at 11 29 18 PM

This PR fixes #120936

### CODE REVIEW

**Comment by isidorn** [comment]
@nrayburn-tech this is a great PR, and works nicely -> approved.

Let's just get @aeschli feedback on the color names. We are introducing:
`debugInline.decorationBackground` and `debugInline.decorationForeground`
What do you think about those?

fyi @weinand

**Review by isidorn — APPROVED** [review]

File: src/vs/workbench/contrib/debug/browser/debugEditorContribution.ts [hunk 1]
```diff
@@ -45,13 +45,29 @@ import { Event } from 'vs/base/common/event';
...
+	hc: '#ffffff80'
+}, nls.localize('debugInline.decorationForeground', "Color for the debug inline value text."));
+
+export const debugInlineBackground = registerColor('debugInline.decorationBackground', {
+	dark: '#ffc80033',
+	light: '#ffc80033',
+	hc: '#ffc80033'
+}, nls.localize('debugInline.decorationBackground', "Color for the debug inline value background."));
+
+
```
Comment (isidorn) [inline src/vs/workbench/contrib/debug/browser/debugEditorContribution.ts:69]: Please remove these two newlines you added - they are not needed

**Comment by weinand** [comment]
@isidorn Looks good to me.

**Comment by isidorn** [comment]
After discussing with @aeschli we landed on the following names:
`editor.inlineValuesBackground` 
`editor.inlineValuesForeground`

The biggest discussion was around if `debug` should be in the name or not. At the end we landed on NO because:
* there is inlineValueProvider in the VS Code API and it is in the `language` namespace. Thus it does not have to be connected to a debugging
* There is already `stackFrameHighlight` and `focusedStackFrameHighlight` colors which do not use the word `debug`

@nrayburn-tech can you please update the color names.
@weinand let me know if the names work for you

**Review by isidorn — CHANGES_REQUESTED** [review]
Please just update color name

**Comment by nrayburn-tech** [comment]
@isidorn updated the names.  I left the descriptions the same as I think debug is the only place that uses them, otherwise it could be more generalized like the name is.  Let me know if you want anything else changed.

**Comment by isidorn** [comment]
Great, thanks a lot for updating the names. This PR looks good -> merging in ☀️ 👏

**Review by isidorn — APPROVED** [review]

**Pull request merged**